script.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var citySelected = document.getElementById("city-select");

```

Comment:
(修正不要)
varは廃止して基本const、必要に応じてletで宣言するようにしましょう

```suggestion
const citySelected = document.getElementById("city-select");
```